content/company/values.md

```diff
@@ -83,11 +83,11 @@ Being open and transparent builds trust with our customers and within our team.
-| Internal to Sourcegraph | Information that is not public for specific reasons, but is shared with Sourcegraph teammates.                                         | Public Slack channels, Google Docs accessible to everyone at Sourcegraph, internal [Google Groups](https://groups.google.com/all-groups)                                                                    | Board slides, company financial info, customer information, open security investigations |
-| Private                 | Information that is neither public nor internal to Sourcegraph for specific reasons, but is shared only with a specific set of people. | Private Slack channels or DMs, private Google Docs, private emails                                                                                                                                          | Feedback, compensation, and personal information of teammates and candidates             |
+| Level of transparency   | Description                                                                                                                            | Channels                                                                                                                                                                                                    | Examples                                                                                                   |
+| ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------- |
+| Public                  | Anyone outside of Sourcegraph can access this information. **This is the default level of transparency.**                              | [Handbook](../index.md), [code](https://github.com/sourcegraph/sourcegraph), [issues](https://github.com/sourcegraph/sourcegraph/issues), public Google Docs such as [RFCs](../communication/rfcs/index.md) | Goals, plans, all code, product docs, dev docs, company docs                                               |
+| Internal to Sourcegraph | Information that is not public for specific reasons, but is shared with Sourcegraph teammates.                                         | Public Slack channels, Google Docs accessible to everyone at Sourcegraph, internal [Google Groups](https://groups.google.com/all-groups)                                                                    | Business metrics, board slides, company financial info, customer information, open security investigations |
```

Comment:
This is the actual current metric values from reporting, and also goals with numbers as under this heading?: https://handbook.sourcegraph.com/company/strategy/code-graph#make-the-power-of-our-features-easier-to-find-and-use

(i.e., do not show, for example, that we are trying to increase WAU by 15%)

Reply:
Good question. Personally, I am more comfortable talking about relative increases in things like WAU, but would not be comfortable talking about absolute numbers.

Reply:
I agree that relative increases are more likely to be OK than absolute numbers.

@dadlerj Can your team come up with some guidance around this?

Reply:
Relative vs. absolute would be really nice, clear guidance if that's workable. Something like "You should never publicly state the starting value in a metric. The target is fine as long as it is a %, and fixed increases (such as 'increase by 5') are at your discretion."